### ISSUE TITLE
Feature request: Store finish timestamp of successful completion of JSM

### ISSUE THREAD

**Issue by kinglok** [Jamf-Concepts/Setup-Manager#9]
It would be very useful in creating smart groups based on the enrollment time through JSM if there is a flag file with completion time.

This will give us more granular control where policies can be run after x days of build.

It is suggested that completion time can be added to the flag file - /private/var/db/.JamfSetupEnrollmentDone

**Comment by entusdictus** [comment]
You already can. The flag file has a timestamp on it which can be utilized by an extension attribute. You may use this code snippet as a base for your extension attribute:

`stat -x -t "%Y-%m-%d" "/private/var/db/.JamfSetupEnrollmentDone" | grep "Birth" | awk '{print $2}'`

Of course you can adjust the timestamp and the awk to also include %H:%M:%S, if needed.

**Comment by scriptingosx** [comment]
shorter stat command:

```
stat -f %SB -t %FT%T /private/var/db/.JamfSetupEnrollmentDone
```

**Comment by scriptingosx** [comment]
This has been implemented with [v1.1beta](https://github.com/jamf/Setup-Manager-private/releases/tag/v1.1beta). Please test and let us know if this works for you.
